orca-pipelinetemplate/src/test/groovy/com/netflix/spinnaker/orca/pipelinetemplate/PipelineTemplatePipelinePreprocessorSpec.groovy

```diff
@@ -334,6 +338,16 @@ class PipelineTemplatePipelinePreprocessorSpec extends Specification {
     result.limitConcurrent == false
   }
 
+  def "should render stages dynamically with jinja"() {
```

Comment:
What else could stages be but a String?  Possible to add a test for a non-String case?

Reply:
It's not much else it can be... If it's not a String, then it will be an object (specifically Map), and all the other tests are already verifying this.

Reply:
The PR description says
> if the stages section is a single string

which is why I was thinking about it.

Reply:
Ah, right! The test _is_ actually testing that: https://github.com/spinnaker/orca/pull/4462/files#diff-8a720d0cfc5eace01931edc129f9ad66d30d584c561234cbfce8cd08bef6f308R10
(Note the pipe at the end of the line, which makes the entire block a single, multiline string)

Reply:
Aaah, ok.  Perhaps update the name of the test to mention this?

Reply:
I've updated the test description and added a comment in the yaml file that is being tested